### PR TITLE
Fix documentation errors in environment variables

### PR DESCRIPTION
**Pull request by ayatnkw**
## Changes

- Fix typos and errors and add more details in [environment variables](https://docs.n8n.io/hosting/configuration/environment-variables/) description
- Fix broken links

## Related tickets and issues
Forum: [Several Documentation Errors](https://community.n8n.io/t/several-documentation-errors/43415) 

Partially resolves https://github.com/n8n-io/n8n-docs/issues/1995

### CODE REVIEW

File: docs/hosting/configuration/environment-variables/binary-data.md [hunk 1]
```diff
@@ -11,10 +11,14 @@ hide:
 
 # Binary data environment variables
 
+/// note | File-based configuration
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/binary-data.md:14]: @ayatnkw probably worth making this into a snippet as we're reusing it in multiple places. To do this:

1. Create a file in `_snippets` (in the relevant subfolder)
2. Add the snippet everywhere you want to use the contents:

```
--8<-- "_snippets/<my-folder>/<my-file.md>"

File: docs/hosting/configuration/environment-variables/logs.md [hunk 2]
```diff
@@ -11,20 +11,24 @@ hide:
...
 | Variable | Type  | Default  | Description |
 | :------- | :---- | :------- | :---------- |
-| `N8N_LOG_LEVEL` | Enum string: `info`, `warn`, `error`, `verbose`, `debug` | `info` | Log output level. |
-| `N8N_LOG_OUTPUT` | Enum string: `console`, `file` | `console` | Where to output logs. Provide multiple values as a comma-seperated list. |
+| `N8N_LOG_LEVEL` | Enum string: `info`, `warn`, `error`, `verbose`, `debug` | `info` | Log output level. Refer to [Log levels](/hosting/logging-monitoring/logging/#log-levels) for details. |
+| `N8N_LOG_OUTPUT` | Enum string: `console`, `file` | `console` | Where to output logs. Provide multiple values as a comma-separated list. |
 | `N8N_LOG_FILE_COUNT_MAX` | Number | `100` | Max number of log files to keep. |
 | `N8N_LOG_FILE_SIZE_MAX` | Number | `16` | Max size of each log file in MB. |
-| `N8N_LOG_FILE_LOCATION` | String | `file` | Log file location. Requires N8N_LOG_OUTPUT set to `file`. |
+| `N8N_LOG_FILE_LOCATION` | String | `<n8nFolderPath>/logs/n8n.log` | Log file location. Requires N8N_LOG_OUTPUT set to `file`. |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/logs.md:29]:
```suggestion
| `N8N_LOG_FILE_LOCATION` | String | `<n8n-directory-path>/logs/n8n.log` | Log file location. Requires N8N_LOG_OUTPUT set to `file`. |
```

Comment (StarfallProjects) [reply docs/hosting/configuration/environment-variables/logs.md:29]: I realise this is a convention I've been following without specifying it in the style guide. I've now added it.

File: docs/hosting/configuration/environment-variables/logs.md [hunk 3]
```diff
@@ -11,20 +11,24 @@ hide:
...
 | `DB_LOGGING_ENABLED` | Boolean | `false` | Whether to enable database-specific logging. |
-| `DB_LOGGING_OPTIONS` | Enum string: `query`, `error`, `schema`, `warn`, `info`, `log`  | `error` | Database log output level. To enable all logging, specify `all`. |
+| `DB_LOGGING_OPTIONS` | Enum string: `query`, `error`, `schema`, `warn`, `info`, `log`  | `error` | Database log output level. To enable all logging, specify `all`. Refer to [typeorm logging options](https://orkhan.gitbook.io/typeorm/docs/logging#logging-options){:target=_blank .external-link} |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/logs.md:31]:
```suggestion
| `DB_LOGGING_OPTIONS` | Enum string: `query`, `error`, `schema`, `warn`, `info`, `log`  | `error` | Database log output level. To enable all logging, specify `all`. Refer to [TypeORM logging options](https://orkhan.gitbook.io/typeorm/docs/logging#logging-options){:target=_blank .external-link} |
```

Comment (StarfallProjects) [reply docs/hosting/configuration/environment-variables/logs.md:31]: Check the website: the brand name is spelled TypeORM

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See inline comments

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See comment in discussion - snippet location

File: docs/hosting/configuration/environment-variables/binary-data.md (hunk 1 above)
Comment (ayatnkw) [reply docs/hosting/configuration/environment-variables/binary-data.md]: moved the snippet to self-hosting directory @StarfallProjects

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**